server/Controllers/authController.js

```diff
@@ -0,0 +1,25 @@
+const bcrypt = require('bcrypt');
+const model = require('../Models/model.js');
+
+const authController = {};
+
+authController.register = async (req, res, next) => {
+  const { username, password, email } = req.body;
+  const queryText = `INSERT INTO User_credentials (username,password,email) VALUES ($1,$2,$3)`;
```

Comment:
Nice. this change makes sense because we expect the user to register with username, password, AND email. Moving email information from Users to User_credentials is also a better UX, since it's probably a bad idea to display emails publicly anyway.